src/katello/agent/katelloplugin.py

```diff
@@ -117,10 +117,11 @@ def setup_plugin():
     cfg = plugin.cfg()
     rhsm_conf = Config(RHSM_CONFIG_PATH)
     certificate = ConsumerIdentity.read()
+    cfg.messaging.cacert = rhsm_conf['rhsm']['repo_ca_cert'] % rhsm_conf['rhsm']
```

Comment:
Unfortunately `Config` doesn't do it, which is why I'm doing the substitution with `rhsm_conf['rhsm']`

I wonder if we can't get this hooked up to Jenkins? Not urgent, I ran the tests locally and they pass.  I need to get a jenkins account one of these days...